### PR TITLE
Fix the cluster-local docs to be consistent

### PR DESCRIPTION
**Pull request by vagababov**
Use same terms throughout and same capitalization.

/assign @abrennan89

### CODE REVIEW

File: docs/serving/cluster-local-route.md [hunk 1]
```diff
@@ -27,18 +27,18 @@ inside the cluster:
...
 
-To label the KService:
+To label the Knative service:
 
 ```shell
 kubectl label kservice ${KSVC_NAME} networking.knative.dev/visibility=cluster-local
 ```
 
-To label a route when you used Route directly without KService:
+To label a route when the route was used directly without Knative service:
```
Comment (abrennan89) [inline docs/serving/cluster-local-route.md:41]:
```suggestion
To label a route when the route is used directly without a Knative service:
```

**Review by abrennan89 — COMMENTED** [review]
One minor comment but otherwise looks good! Thank you @vagababov :slightly_smiling_face:

**Comment by vagababov** [comment]
Done

**Comment by abrennan89** [comment]
/lgtm
/approve

Looks like the same cla issue. This thing is the bane of my existence. Only solution I know is to push again or get someone from Google to manually fix the label.

**Comment by vagababov** [comment]
OK let me hard push it to fix.

**Comment by vagababov** [comment]
For some reason, I can't just edit labels in this repo :/

**Comment by vagababov** [comment]
OK force pushed, CLA should be ok now.

**Pull request merged**